DisplayOptionsPanel.c

```diff
@@ -117,11 +117,17 @@ DisplayOptionsPanel* DisplayOptionsPanel_new(Settings* settings, ScreenManager*
    Panel_add(super, (Object*) CheckItem_newByRef("Update process names on every refresh", &(settings->updateProcessNames)));
    Panel_add(super, (Object*) CheckItem_newByRef("Add guest time in CPU meter percentage", &(settings->accountGuestInCPUMeter)));
    Panel_add(super, (Object*) CheckItem_newByRef("Also show CPU percentage numerically", &(settings->showCPUUsage)));
+   #ifndef HTOP_DARWIN
```

Comment:
Should use `defined` due to `#elif defined()`
```suggestion
   #if !defined(HTOP_DARWIN)
```